packages/rspack/tests/Stats.test.ts

```diff
@@ -177,4 +177,64 @@ describe("Stats", () => {
 		"
 	`);
 	});
+
+	it("should have reasons", async () => {
+		const stats = await compile({
+			context: __dirname,
+			entry: "./fixtures/main5.js",
+			stats: { reasons: true }
```

Comment:
```suggestion

```
this line is not necessary if using `stats.toJson({ all: false, reasons: true })`

Reply:
Seems `reasons` option is not work, and I can't find any operation for `options.reasons` in `toJson` function.

Reply:
> Seems `reasons` option is not work

Just found it's a bug, we may fix it in the following PR.